var/spack/repos/builtin/packages/simmetrix-simmodsuite/package.py

```diff
@@ -0,0 +1,224 @@
+         'msparallelmesh': ['72e4bb6d60b00c82d39642f75ed4a665', 'parallelmesh']
+      },
+      'docs': {
+         'FieldSim': ['7eee54361cce486b268a8fd78e8eddf4', 'base'],
+         'GeomSim': ['a0eca9ec56a31682dc817f6aa6d73823', 'base'],
+         'MeshSim': ['4c360a6ad6f47d674d08f84cdd272df4', 'base'],
+         'MeshSimAdapt': ['d6f3f06a1a43b5003fd302dd219a55df', 'base'],
+         'GeomSimAcis': ['24ac7f8516a65ce487c2b31d39dbe407', 'acis'],
+         'GeomSimParasolid': ['b1914bbcaa2d91808eea364fdaba9f4a', 'parasolid'],
+         'GeomSimDiscrete': ['cee3f1e7dc0b3aeafeeafcb59eecbf40', 'discrete'],
+         'GeomSimAbstract': ['e3c02ab678c6aeada01c83f059e575bb', 'abstract'],
+         'GeomSimAdvanced': ['fabf3b53dce09a5082b084e26e8a80ef', 'advmodel'],
+         'GeomSimImport': ['16cc4bad9e6e84d97583ac84e5968954', 'import'],
+         'GeomSimVoxel': ['d9e10e2cb89e215534c2c04538994233', 'voxel'],
+         'MeshSimAdvanced': ['2623d6edce07103ebbac4b7e9b38332d', 'adv'],
+         'ParallelMeshSimAdapt': ['f82a8f60760503f524714634ffe85cfb', 'paralleladapt'],
+         'ParallelMeshSim': ['17596e3618f91c8d4a8be750051271a5', 'parallelmesh']
+      }
+    }
+    ]
+    return releases
+
+
+def simmetrix_getKernel():
+    """only supporting the linux libraries"""
+    return "linux"
+
+
+def simmetrix_getWordSz():
+    """only supporting 64b libraries"""
+    return "64"
+
+
+def simmetrix_makeComponentUrl(name):
+    """only supporting the linux libraries"""
+    prefix = "file://{0}/".format(os.getcwd())
+    suffix = "-" + simmetrix_getKernel() + simmetrix_getWordSz() + '.tgz'
+    return prefix + name + suffix
+
+
+def simmetrix_makeDocUrl(name):
+    """doc zip files are not os/arch specific"""
+    prefix = "file://{0}/".format(os.getcwd())
+    suffix = '.zip'
+    return prefix + name + suffix
+
+
+def simmetrix_resource(name, url, md5, condition):
+    # The tarballs/zips each have the same directory structure.  Because of
+    # this, and the bug in spack described here:
+    # https://github.com/spack/spack/pull/3553#issuecomment-391424244
+    # , they cannot be expanded into the source root directory.
+    # Once this is fixed the 'destination=name' argument can be removed.
+    resource(
+      name=name,
+      url=url,
+      md5=md5,
+      destination=name,
+      when=condition
+    )
+
+
+def simmetrix_copytree(src, dst):
+    dir_util.copy_tree(src, dst)
+
+
+class SimmetrixSimmodsuite(Package):
+    """Simmetrix' Simulation Modeling Suite is a set of component software
+    toolkits that allow developers to easily implement geometry-based
+    simulation applications.
+    Each component of the Simulation Modeling Suite is designed to address
+    specific capabilities:
+    | MeshSim - automatic mesh generation
+    | FieldSim - simulation data management
+    | GeomSim - direct, untranslated access to geometry from a wide variety
+    of sources
```

Comment:
Run `spack info` and see how this is formatted. Newline characters are stripped out, so I don't think this will look good.

Reply:
It's not too bad.  The vertical bars work OK for delimiters:

```
Description:
    Simmetrix' Simulation Modeling Suite is a set of component software
    toolkits that allow developers to easily implement geometry-based
    simulation applications. Each component of the Simulation Modeling Suite
    is designed to address specific capabilities: | MeshSim - automatic mesh
    generation | FieldSim - simulation data management | GeomSim - direct,
    untranslated access to geometry from a wide variety of sources

Homepage: http://www.simmetrix.com/products/SimulationModelingSuite/main.html
```